### PR TITLE
Correct IntegrationConfig, and make the Tenant Operator name consiste…

### PR DESCRIPTION
**Pull request by MahnoorAsghar**
…nt in terms of case

### CODE REVIEW

**Comment by stakater-user** [comment]
@MahnoorAsghar Image is available for testing. `docker pull docker-stakater-nexus.apps.prod.clusters.stakater.io/stakater/stakaterclouddocs:SNAPSHOT-PR-59-362d4e27`

**Comment by stakater-user** [comment]
@MahnoorAsghar Image is available for testing. `docker pull docker-stakater-nexus.apps.prod.clusters.stakater.io/stakater/stakaterclouddocs:SNAPSHOT-PR-59-4406ff55`

**Comment by stakater-user** [comment]
@MahnoorAsghar Image is available for testing. `docker pull docker-stakater-nexus.apps.prod.clusters.stakater.io/stakater/stakaterclouddocs:SNAPSHOT-PR-59-2b6f7e0b`

**Comment by stakater-user** [comment]
@MahnoorAsghar Image is available for testing. `docker pull docker-stakater-nexus.apps.prod.clusters.stakater.io/stakater/stakaterclouddocs:SNAPSHOT-PR-59-45443c3e`

**Comment by stakater-user** [comment]
@MahnoorAsghar Image is available for testing. `docker pull docker-stakater-nexus.apps.prod.clusters.stakater.io/stakater/stakaterclouddocs:SNAPSHOT-PR-59-17f1f714`

File: .github/workflows/pull_request.yaml [hunk 1]
```diff
@@ -24,7 +24,6 @@ jobs:
       - name: Spell check
         uses: errata-ai/vale-action@v2.0.1
         with:
-          styles: https://github.com/errata-ai/write-good/releases/latest/download/write-good.zip
```
Comment (karl-johan-grahn) [inline .github/workflows/pull_request.yaml]: don't remove this

File: .vale.ini [hunk 2]
```diff
@@ -1,7 +1,7 @@
-StylesPath = "vocabulary/styles"
```
Comment (karl-johan-grahn) [inline .vale.ini]: no need to change this

**Comment by stakater-user** [comment]
@MahnoorAsghar Image is available for testing. `docker pull docker-stakater-nexus.apps.prod.clusters.stakater.io/stakater/stakaterclouddocs:SNAPSHOT-PR-59-ed8eaceb`

**Comment by stakater-user** [comment]
@MahnoorAsghar Image is available for testing. `docker pull docker-stakater-nexus.apps.prod.clusters.stakater.io/stakater/stakaterclouddocs:SNAPSHOT-PR-59-e298be8d`

**Comment by stakater-user** [comment]
@MahnoorAsghar Image is available for testing. `docker pull docker-stakater-nexus.apps.prod.clusters.stakater.io/stakater/stakaterclouddocs:SNAPSHOT-PR-59-28a7c525`

**Review by rasheedamir — APPROVED** [review]

**Comment by stakater-user** [comment]
@rasheedamir Image is available for testing. `docker pull docker-stakater-nexus.apps.prod.clusters.stakater.io/stakater/stakaterclouddocs:v0.0.37`

**Pull request merged**